### PR TITLE
[15.0][ADD] stock_move_purchase_uom

### PR DESCRIPTION
**Pull request by MarinaAForgeFlow**

### CODE REVIEW

File: stock_move_purchase_uom/models/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from odoo import api, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    @api.onchange("product_id", "picking_type_id")
+    def _onchange_product_id(self):
+        super()._onchange_product_id()
```
Comment (JordiMForgeFlow) [inline stock_move_purchase_uom/models/stock_move.py]:
```suggestion
        res = super()._onchange_product_id()
```

File: stock_move_purchase_uom/models/stock_move.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        if self.picking_type_id and self.picking_type_id.use_purchase_uom:
+            self.product_uom = self.product_id.with_context(
+                lang=self._get_lang()
+            ).uom_po_id.id
+
+        return
```
Comment (JordiMForgeFlow) [inline stock_move_purchase_uom/models/stock_move.py]:
```suggestion
        return res
```

File: stock_move_purchase_uom/models/stock_move.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+from odoo import api, models
```
Comment (LoisRForgeFlow) [inline stock_move_purchase_uom/models/stock_move.py:3]: You should add the copyrights in all python files excluding __init__'s

**Review by LoisRForgeFlow — COMMENTED** [review]
LGTM

Can you squash all your commits?

**Comment by rousseldenis** [comment]
My question regarding this is: isn't it possible to use product packaging in this case ?

**Comment by LoisRForgeFlow** [comment]
> My question regarding this is: isn't it possible to use product packaging in this case ?

We also evaluated that option, but in this case UoM was really embedded in many processes and therefore this proposal was easier to introduce.

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

/ocabot merge nobump

**Pull request merged**